### ISSUE TITLE
Migrate JSON Merging to its own package

### ISSUE THREAD

**Issue by jamietanna** [oapi-codegen/runtime#16]

**Comment by jamietanna** [comment]
No, if the issue hasn't been updated, there's no updates :) Is there any reason you need this?

**Comment by simonklb** [comment]
I'm creating a tiny API client and generating some of the code using oapi-codegen. While it's not a big deal my go.sum would be cut in half by adopting the code.
